web/src/rendered_markdown.ts

```diff
@@ -346,7 +345,13 @@ export const update_elements = ($content: JQuery): void => {
             const $inline_img_thumbnail = $(this);
             let thumbnail_name = thumbnail.preferred_format.name;
             if ($inline_img_thumbnail.attr("data-animated") === "true") {
-                thumbnail_name = thumbnail.animated_format.name;
+                if (user_settings.web_animate_image_previews === "always") {
+                    thumbnail_name = thumbnail.animated_format.name;
+                } else {
+                    $inline_img_thumbnail
+                        .closest(".message_inline_image")
+                        .addClass("message_inline_animated_image");
+                }
```

Comment:
Why the name `message_inline_animated_image`? It seems like this is used for the case where the thing is not currently animated, but can be animated.

Reply:
basically it would be added when the animated images are is not currently playing i.e., to get the play icon on the image. So, yeah it is definitely not a good name. So I guess I'll update it to `message_inline_animated_image_paused`.

Reply:
Yeah, or `message_inline_animated_image_still` might be even better, since `still` is the term we use for the non-animated preview files for animated images.